core/src/main/java/jenkins/model/item_category/ItemCategory.java

```diff
@@ -47,8 +47,8 @@
      * See <a href="https://issues.jenkins.io/browse/JENKINS-36593">JENKINS-36593</a> for more info.
      */
     @Restricted(NoExternalUse.class)
-    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "TODO needs triage")
-    public static int MIN_TOSHOW = 1;
+    @RestrictedSince("2.14")
+    public static final int MIN_TOSHOW = 1;
```

Comment:
Since this is restricted, it is safe to mark it final, as no plugins will inline the value during compilation. I verified that there were no references to this in the `jenkinsci` GitHub organization or in any plugin binaries.